Exiled.API/Features/Door.cs

```diff
@@ -144,28 +146,52 @@ public Vector3 Position
         }
 
         /// <summary>
-        /// Gets a value indicating whether or not SCP-106 can walk through the door.
+        /// Gets or sets a value indicating whether or not SCP-106 can walk through the door.
         /// </summary>
-        public bool AllowsScp106 => Base is IScp106PassableDoor door && door.IsScp106Passable;
+        public bool AllowsScp106
+        {
+            get => Base is IScp106PassableDoor door && door.IsScp106Passable;
+            set
+            {
+                if (Base is IScp106PassableDoor door)
+                    door.IsScp106Passable = value;
+            }
+        }
 
         /// <summary>
-        /// Gets a value indicating whether or not the door is locked.
+        /// Gets or sets a value indicating whether or not the door is locked.
         /// </summary>
-        public bool IsLocked => DoorLockType > 0;
+        public bool IsLocked
+        {
+            get => DoorLockType > 0;
+            set
+            {
+                if (value)
+                    ChangeLock(DoorLockType.AdminCommand);
+                else
+                    Unlock();
+            }
```

Comment:
I don't thinks it's a good idea to put a setter for this

Reply:
Why?

Reply:
because people should use them self UnLock / ChangeLock(Reason)
and if someone just does
IsLocked = true;
IsLocked = false: it's would remove other lock like Scp079 or Warhead

and if we make it only unlock AdminCommand people will ask why it's do no unlock Scp079 or Warhead for example